aconcli/service/aconclient_http.go

```diff
@@ -0,0 +1,465 @@
+package service
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"mime/multipart"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+)
+
+const (
+	endpointBlob     = "/api/v1/blob"
+	endpointManifest = "/api/v1/manifest"
+	endpointFinalize = "/api/v1/manifest/finalize"
+	endpointStart    = "/api/v1/container/start"
+	endpointExec     = "/api/v1/container/exec"
+	endpointInspect  = "/api/v1/container/inspect"
+	endpointReport   = "/api/v1/container/report"
+	endpointKill     = "/api/v1/container/kill"
+	endpointRestart  = "/api/v1/container/restart"
+
+	fieldManifest  = "manifest"
+	fieldSig       = "sig"
+	fieldCert      = "cert"
+	fieldImgeId    = "image_id"
+	fieldMissLayer = "missing_layers"
+	fieldAlg       = "alg"
+	fieldBlob      = "data"
+	fieldEnvs      = "envs"
+	fieldConId     = "container_id"
```

Comment:
We've been using `cid` for "container ID". Suggest `fieldCid = "cid"` or `fieldContainerID = "container_id"`.

Reply:
Fixed